### PR TITLE
[CIS-60] feature(web): Home View - About Section

### PR DESCRIPTION
**Pull request by dustinjack99**
# JIRA Ticket:

Please include a link to the JIRA ticket that this pull request is related to:
https://mammoth-studios.atlassian.net/browse/CIS-60

## Description

Implementation of the Home About section of the Home View.

## Type of Change

- [ ] Bug fix
- [X] New feature
- [ ] Refactor

### About Section
Screenshot 2024-03-02 at 7 07 09 PM

### Credits Section
Screenshot 2024-03-02 at 7 40 44 PM

### CODE REVIEW

**Comment by ThatZiro** [comment]
Just some minor bugs I noticed with the responsiveness.
 
SVG Corner is cutting into text :
image

Text elements are bleeding outside of box :
image

Should consider switching to a Column on smaller screens :
image

File: front-end/src/domains/home/components/home-about/home-about.component.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      <div className="home-about-credits">
+        <h2>Credits</h2>
+        <div className="home-about-credits-container">
+          {credits.map(({ department, roles }, index) => (
+            <div key={index} className="home-about-credits-department">
+              <h3>{department}</h3>
+              <ul>
+                {roles.map(({ name, role }, index) => (
+                  <li key={index}>
+                    {name} - {role}
```
Comment (ThatZiro) [inline front-end/src/domains/home/components/home-about/home-about.component.tsx:27]: Change this to use a Template String to add some extra readability
```
`${name} - ${role}`
```

**Review by ThatZiro — DISMISSED** [review]
### Great Work, Dustin!

I want to commend you on the excellent work you've done on this ticket. Your attention to detail and commitment to quality shine through in every aspect of the codebase!

### A Few Notes:

**Code Readability:**
Kudos, Dustin! Your code is well-structured and easy to follow. However, for even better readability, consider implementing a minor change.

**Responsiveness:**
I've added comments regarding responsiveness. It would be beneficial to include considerations for various devices and screen sizes.

**File Naming:**
Noticed a discrepancy in file naming conventions. Specifically, `home-about.constant.ts` and `home-cards.constant.ts` appear to serve similar functions. Shouldn't they have consistent naming conventions, including file extensions?

Once again, great job! Just make these little fixes, and you're good to merge, Awesome work!

File: front-end/src/domains/home/components/home-about/home-about.component.tsx (hunk 1 above)
Comment (dustinjack99) [reply front-end/src/domains/home/components/home-about/home-about.component.tsx:27]: Question - why is this necessary when React does this out of the box? 🤔

**Comment by dustinjack99** [comment]
@ThatZiro - pushed up some changes to fix responsiveness in a few place. 

Didn't get all the fixes, but since we're still in lo-fi specs I'll save the fine-tuning for when we're more confident what content we're working with 👍

**Pull request merged**